Kiosk/Admin/AdminLogViewController.swift

```diff
@@ -4,7 +4,10 @@ class AdminLogViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        textView.text = try? NSString(contentsOf: logPath(), encoding: String.Encoding.ascii.rawValue) as String
+        textView.text = try? NSString(
+            contentsOf: logPath(),
+            encoding: String.Encoding.ascii.rawValue
+        ) as String
```

Comment:
```swift
        textView.text = try? NSString(contentsOf: logPath(), encoding: String.Encoding.ascii.rawValue) as String
//-----------------------------------------------------------------------------|
        textView.text = try?
            NSString(
                contentsOf: logPath(),
                encoding: String.Encoding.ascii.rawValue
            ) as String
```

maybe?

Reply:
I haven't made up my mind on these situations yet.

---

Generally the following seems to be reasonable if we don't have `as?` or `try?` around and is also what Prettier JS does:

```
let foo = Bar(
   baz: 123
)
```

I just changed `as?` to bind stronger to the right, so I do this

```
fooFooFooFoo
    as? Bar
```

instead of 

```
fooFooFooFoo as?
    Bar
```

---

So I'd rather keep `try` on the same line as the beginning of the call expr, so that would mean:

```
textView.text = try? NSString(
        contentsOf: logPath(),
        encoding: String.Encoding.ascii.rawValue
    ) as String
```

This looks a bit odd by also gives `try` more prominence. Might look weirder if you have a long chain though:

```
fooFooFooFoo = try? fooFooFooFoo.
        .fooFooFooFoo()
        .fooFooFooFoo()
```